### PR TITLE
Sebastian changes

### PR DESCRIPTION
**Pull request by SebastianQJ**
Http updated reservations

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package com.outofbounds.empire.Reservation.Controllers;
+
+import com.outofbounds.empire.Reservation.Models.Reservation;
+import com.outofbounds.empire.Reservation.Repositories.ReservationRepository;
+import com.outofbounds.empire.Showings.Repositories.ShowingRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        return reservationRepository.findAll();
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "/movies/{id}")
+    public @ResponseBody
+    Reservation reservation(@PathVariable int id) {
+        return reservationRepository.findById(id);
+    }
+
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    public @ResponseBody
+    Reservation addReservation(
+            @RequestParam (required = true) int seats,
+            @RequestParam (required = true) String phoneNumber,
+            @RequestParam (required = true) int showroomid ) {
+
+        Reservation reservation = new Reservation(showingRepository.findById(showroomid),seats,phoneNumber);
+        reservationRepository.save(reservation);
+        return reservation;
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    @RequestMapping(method = RequestMethod.DELETE, value = "/movies/{id}")
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java]: add spacing

File: src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    boolean deleteReservation(
+            @PathVariable int id){
+        Reservation reservation =  reservationRepository.findById(id);
+        if(reservation == null){
+            return false;
+        }
+        reservationRepository.delete(reservation);
+        return true;
+    }
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Controllers/ReservationController.java]: remove several spacings

File: src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
+package com.outofbounds.empire.Reservation.Models;
+
+import com.outofbounds.empire.Showings.Models.Showing;
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    private int id;
+
+    @ManyToOne()
+    @JoinColumn(name = "showing.id", nullable = false)
+    private Showing showing;
+    @Column(name = "seat_number")
+    private int seat;
+    @Column(name = "customers")
+    private String phoneNumber;
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java:21]: remove several spacings

File: src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    public void setSeat(int seat) {
+        this.seat = seat;
+    }
+
+
+
+
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java]: remove several spacings

File: src/main/java/com/outofbounds/empire/Reservation/Repositories/ReservationRepository.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
+package com.outofbounds.empire.Reservation.Repositories;
+
+import com.outofbounds.empire.Reservation.Models.Reservation;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Repositories/ReservationRepository.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Repositories/ReservationRepository.java [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+import java.util.List;
+
+@Transactional
+@Repository
+public interface ReservationRepository extends CrudRepository<Reservation, Integer> {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Repositories/ReservationRepository.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 11]
```diff
@@ -1,10 +1,12 @@
 package com.outofbounds.empire.Showings.Models;
 
 import com.outofbounds.empire.Movies.Models.Movie;
+import com.outofbounds.empire.Reservation.Models.Reservation;
 import com.outofbounds.empire.Showrooms.Models.Showroom;
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: remove spacing

**Review by AnderskdKnudsen — COMMENTED** [review]
Only spacing issues. Nice job, guys! Holy shit, best code today

**Comment by SebastianQJ** [comment]
Changes done

**Review by AnderskdKnudsen — APPROVED** [review]
Great!

**Pull request merged**